### PR TITLE
Fixes and testing

### PR DESCRIPTION
**Pull request by colefiscus**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [ ] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
> This PR makes styling changes for mobile viewing. Initially it was going to be a testing branch as well, but we've figured out a better way to do that.

## *Where should the reviewer start?*
*Provide guidance for where the reviewer should start &/or how to manually test.*
> All the files changed have independent, but clear changes. No single great place to start.

## *What are the relevant tickets?*
Closes _
> N/A

## *Concerns / Bugs:*
*Additional context, concerns, issues, or bugs in this branch? If so describe them.*
> The one issue I found which could be fixed if necessary is - because HomeView.css and ProfileView.css both have the same rules, or at least they both have a rule for the .feed-container selector, they both have to be edited for a change to occur because one is overriding the other.

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have used tests to prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my change

### CODE REVIEW

**Review by christina-cook — APPROVED** [review]
Looks good!

File: src/components/FormModal/FormModal.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const FormModal = ({ user, toggleFormModal }) => {
 
   return (
 
-    <article className="form-modal">
+    // <article className="form-modal">
```
Comment (dietza) [inline src/components/FormModal/FormModal.js]: Good catch - looks like this wrapping article isn't really necessary. Removing that now to push up.

File: public/manifest.json [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
     },
     {
-      "src": "logo192.png",
+      "src": "favicon.ico",
       "type": "image/png",
       "sizes": "192x192"
     },
     {
-      "src": "logo512.png",
+      "src": "favicon.ico",
```
Comment (dietza) [inline public/manifest.json:16]: Hopefully this update allows the PWA icon to appear as our chosen one!

**Review by dietza — APPROVED** [review]
Updated icons in the manifest.json file

**Pull request merged**